armada-ths/ais
# Removed CSRF verification for PUT request

Exempted CSRF verification on /api/student_profile PUT method, so it's accepts naked payload now.

to test this:
* run tests
* if you can (I used a REST console extension for chrome) create a manual PUT request to localhost:8000/api/student_profile?student_id={AN_ID} and a json payload: {"nickname":"{A_NICKNAME}"} and you should get the same json back. (And following GET calls to the same ID should also give the same result)

Sidenote: without CSRF verification, we open ourselves to possible attacks, so we probably want some authentication in the near future.
